### PR TITLE
Add basic test

### PR DESCRIPTION
**Pull request by jknoxville**
I was planning to add some extra features to this, so just started off by writing a test to make it easier to develop.

### CODE REVIEW

File: TestFile.java [hunk 1]
```diff
@@ -0,0 +1,5 @@
+package com.remark;
```
Comment (kevin940726) [inline TestFile.java]: I would prefer to put this file into a directory, maybe `__fixtures__`? Also, I would prefer to just use a simple `js` file like shown in the README. The file name could also be the same.

File: package.json [hunk 2]
```diff
@@ -25,14 +25,23 @@
     "gatsby-plugin"
   ],
   "dependencies": {
+    "remark": "^13.0.0",
```
Comment (kevin940726) [inline package.json]: I don't think we need `remark` as a dependency, could instead be a `devDependency`.

File: package.json [hunk 3]
```diff
@@ -25,14 +25,23 @@
...
     "to-gatsby-remark-plugin": "^0.1.0",
     "unist-util-visit": "^2.0.1"
   },
   "devDependencies": {
+    "import-local": "^3.0.2",
```
Comment (kevin940726) [inline package.json]: Why do we need this?

File: package.json [hunk 4]
```diff
@@ -25,14 +25,23 @@
...
   },
   "prettier": {
     "singleQuote": true,
     "trailingComma": "es5"
+  },
+  "scripts": {
+    "test": "jest"
+  },
+  "installConfig": {
+    "pnp": true
```
Comment (kevin940726) [inline package.json]: I'm actually think we could just switch to `npm` and disable lock file, but let's do it in another PR.

File: package.json [hunk 5]
```diff
@@ -25,14 +25,23 @@
...
   "devDependencies": {
+    "import-local": "^3.0.2",
+    "jest": "^26.6.3",
     "prettier": "1.19.1"
   },
   "prettier": {
     "singleQuote": true,
     "trailingComma": "es5"
+  },
+  "scripts": {
```
Comment (kevin940726) [inline package.json]: Nitpick: it's very minor and just my personal preference, could we move this above `dependencies`?

File: test.js [hunk 6]
```diff
@@ -0,0 +1,28 @@
+const codeImport = require('./index.js');
```
Comment (kevin940726) [inline test.js]: Prefer to just import from `./`, so that it could load from `package.json` directly for the sake of e2e testing.

```suggestion
const codeImport = require('./');
```

**Review by kevin940726 — CHANGES_REQUESTED** [review]
Thank you so much for this! Just some minor nitpicks :).

**Comment by jknoxville** [comment]
Glad you like it! I think that's it all taken care of now.

File: test.js [hunk 7]
```diff
@@ -0,0 +1,24 @@
...
+\`\`\`
+`;
+
+test('Basic file import', () => {
+  expect(
+    remark()
+      .use(codeImport, {})
+      .processSync({
+        contents: input,
+        path: __dirname + '/say-hi.js',
```
Comment (kevin940726) [inline test.js]: I think this is wrong, this should be the path of the markdown file. Maybe just `test.md`? And prefer to use standard `path` library to create path string.

```suggestion
        path: path.join(process.cwd(), 'test.md'),
```

**Review by kevin940726 — CHANGES_REQUESTED** [review]

**Review by kevin940726 — APPROVED** [review]
Thanks again!
